src/sass/partials/_logo.scss

```diff
@@ -0,0 +1,12 @@
+.logo {
+  &--size {
+    @media screen and (min-width: 768px) and (max-width: 1279px) {
```

Comment:
@media screen and (min-width: 768px) to wystarczy bez tego drugiego media, tłumaczyłem Ci ostatnio co się stanie jak komuś się wyświetli w przedziale od 1279px do 1280px czasem niektórzy mają 1279.59px i już nam nie zadziała ten media i następny też nie bo będzie od 1280px, także w tym przypadku normalny wymiar jest mobilny i pierwsze media z min wind 768px a drugie z min-width 1280px tego max-width tutaj usun